### PR TITLE
fix(BPModLoader): Loading on slow dedicated servers

### PR DESCRIPTION
**Pull request by localcc**
**Description**

Fixes loading on slow dedicated servers where UE4SS initializes too
late.

Fixes # (issue) (if applicable)

**Type of change**

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)

**How Has This Been Tested?**

Please describe the tests that you ran to verify your changes. Provide instructions so reviewers can reproduce. Please also list any relevant details for your test configuration.

**Checklist**

- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [x] I have added the necessary description of this PR to the changelog, and I have followed the same format as other entries.
- [x] Any dependent changes have been merged and published in downstream modules.

**Screenshots (if appropriate)**

**Additional context**

### CODE REVIEW

File: assets/Mods/BPModLoaderMod/Scripts/main.lua [hunk 1]
```diff
@@ -262,6 +262,11 @@ RegisterLoadMapPostHook(function(Engine, World)
     LoadMods(World:get())
 end)
 
+local ExisitngActor = FindFirstOf("Actor")
+if ExisitngActor ~= nil and ExistingActor:IsValid() then
+    LoadMods(ExistingActor:GetWorld())
+end
```
Comment (UE4SS) [inline assets/Mods/BPModLoaderMod/Scripts/main.lua:268]:
```suggestion
local ExistingActor = FindFirstOf("Actor")
if ExistingActor ~= nil and ExistingActor:IsValid() then
    LoadMods(ExistingActor:GetWorld())
end
```
Typos.

**Review by UE4SS — CHANGES_REQUESTED** [review]

**Comment by UE4SS** [comment]
Are we absolutely sure that the first actor that exists in GUObjectArray always belongs to the correct world ?
I looked at a similar fix but I didn't want to move forward because I didn't have the answer to the above question and I didn't feel like firing up 100 games to test it nor did I feel like looking forever at the UE source code to figure it out.

**Comment by localcc** [comment]
When testing I didn't have any issues, and it doesn't matter if it's not in the correct world yet, because an even for changing worlds will fire which should respawn the mod actors.

**Review by UE4SS — APPROVED** [review]

**Comment by UE4SS** [comment]
Changelog entry is missing, even though the checkbox for it is ticked.

**Comment by Buckminsterfullerene02** [comment]
Why did you have to base this branch off of a really old commit 😂

**Pull request merged**